code/src/Appsinglepage.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
+import 'App.css'
+
+export const App = () => {
```

Comment:
Another suggestion is also to name the component the same as the file, so that it doesn't get confusing! :)